Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -194,6 +194,18 @@ public extension Node where Context == HTML.FormContext {
     static func action(_ url: URLRepresentable) -> Node {
         .attribute(named: "action", value: url.string)
     }
+    
+    /// Define the form’s Media Type.
+    /// - Parameter type: the Media Type the form will be serialized in.
+    static func enctype(_ type: HTMLFormEnctype) -> Node {
```

Comment:
Does the method name have to match the HTML attribute? As otherwise, I think it would be more readable with `encodingType()`.

Reply:
I think that’s a topic of conversation. Let me know if you agree…

Choosing “enctype” matches HTML attributes. This is the right direction, otherwise we’re pretty much putting the HTML standards authorities on trial. I want to avoid something like, “You standards folks chose ‘enctype’, but we’re going to fix that here in our framework.” Ex: replace ‘enctype’ / ‘encodingType’ with ‘h1’ / ‘headerLevelOne’.

This also reduces the need to learn HTML and Plot’s “version” of HTML attributes.

Reply:
Yeah having the names of these low-level node constructors match the names of their HTML equivalents is one of Plot's design goals (which we definitely need better docs on), for exactly the reasons that @mcritz mentions. Then we can build more higher-level components on top of those HTML primitives that use more "Swifty" naming conventions. For example, compare the `.stylesheet()` component to the underlying uses of `href()` and `rel()`.